pkgs/by-name/ma/material-cursors/package.nix

```
@@ -0,0 +1,44 @@
+{ lib, inkscape, xcursorgen, stdenvNoCC, fetchFromGitHub }:
```

Placing these on separate lines gives better diffs
```suggestion
{ lib
, stdenvNoCC
, fetchFromGitHub
, inkscape
, xcursorgen
}:
```